packages/client/src/explore/graphql/explore.graphql

```diff
@@ -0,0 +1,33 @@
+query ExplorePublications($request: ExplorePublicationRequest!, $observerId: ProfileId) {
+  result: explorePublications(request: $request) {
+    items {
+      ... on Post {
+        ...Post
+      }
+
+      ... on Mirror {
+        ...Mirror
+      }
+
+      ... on Comment {
+        ...Comment
+      }
+    }
+    pageInfo {
+      ...CommonPaginatedResultInfo
+    }
+  }
+}
+
+query ExploreProfiles($observerId: ProfileId, $limit: LimitScalar!, $cursor: Cursor) {
```

Comment:
I think in this low level SDK we should just expose all what API accepts, so `ExploreProfilesRequest` in this case